src/types/form.ts

```diff
@@ -147,6 +147,19 @@ export interface IForm {
   emails?: string[] | string
 }
 
+export type FormSettings = Pick<
+  IFormDocument,
```

Comment:
should we "inherit" from `IForm` instead, since we only want a subset of a POJO definition?

Reply:
Problem is (maybe not really a problem) that `IForm` has a bunch of optional fields due to them being default in the form schema. The document type has the most correct shape in that all the default fields are no longer optional (as they should already be created as it is now a document)

Reply:
OK. I'll leave it up to you to decide whether it would help to refactor the types or not.